### PR TITLE
A search vignette

### PR DESCRIPTION
**Pull request by trangdata**
addresses part of #287

### CODE REVIEW

File: vignettes/articles/literature-search.Rmd [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  pages = 1,
+  per_page = 10,
+  verbose = TRUE
+)
+braf_oa |> 
+  show_works(simp_func = identity) |> 
+  select(1:2)
+```
+
+This call performs a search using the OpenAlex API, retrieving the top 10 results for the query "BRAF AND melanoma".
```
Comment (yjunechoe) [inline vignettes/articles/literature-search.Rmd:51]: In light of the discussion on the `sort` option that comes later, I'm not sure what "top 10 results" mean here, when `sort` is unset. Is it top 10 by relevance score? Might be helpful to explicitly describe the default behavior.

**Review by yjunechoe — APPROVED** [review]
LGTM! One minor clarification question (because I personally don't use search much, so I genuinely don't know!)

File: vignettes/articles/literature-search.Rmd (hunk 1 above)
Comment (trangdata) [reply vignettes/articles/literature-search.Rmd:51]: Good point. I will add. And yes by relevance score.

Comment (yjunechoe) [reply vignettes/articles/literature-search.Rmd]: Got it - feel free to merge anytime!

**Pull request merged**